### DISCUSSION TITLE
Store screenshots for product marketing purposes

### DISCUSSION THREAD

**Discussion by Moshyfawn** [vitest-dev/vitest#1500] [Q&A]
I'm trying to configure the testing pipeline to take screenshots of specific app screens _(React app in my case)_ and possibly store them in some storage _(local / remote)_.

The "storing" part might not be related to Vitest specifically, but I'm curious if you have any pointer as to how to approach taking the app screenshots with Vitest.

**Comment by dermoumi** [comment]
Neither vitest nor jsdom or happydom render anything at all. So there's nothing to screenshot.

You probably want to try out cypress or playwright.
